### PR TITLE
fix: wrong text-color class and text contrast on dates page

### PR DESCRIPTION
**Pull request by ihor-romaniuk**
#### Description:
This pull request contains minor fixes with text colors.
- Fixed wrong `text-color` class on dates page
- Increase text color contrast on dates page

|  Before |  After |
|---|---|
|  image |  image |

#### Related Pull Requests
PR to the open-release/palm.master branch: https://github.com/openedx/frontend-app-learning/pull/1222
PR to the open-release/quince.master branch: https://github.com/openedx/frontend-app-learning/pull/1229

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @ihor-romaniuk! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by itsjeyd** [comment]
Hi @ihor-romaniuk, thank you for this contribution!

I'm assuming that the changes are ready for review, but let me know if you were planning on working on them some more.

--

@mattcarter This PR, as well as the corresponding backports to Palm (#1222) and Quince (#1229), are ready for Aurora to have a look.

**Comment by ihor-romaniuk** [comment]
Hi @itsjeyd, this PR is ready for review, thanks.

**Comment by itsjeyd** [comment]
Hey @mattcarter, a friendly reminder to line this PR up for engineering review. Backports for Palm (#1222) and Quince (#1229) are ready to go as well.

File: src/course-home/dates-tab/timeline/badgelist.jsx [hunk 1]
```diff
@@ -38,21 +38,21 @@ function getBadgeListAndColor(date, intl, item, items) {
...
-      className: 'text-black',
+      className: 'text-dark',
     },
     {
       message: messages.pastDue,
       shownForDay: assignments.length && assignments.every(isPastDue),
       shownForItem: x => isLearnerAssignment(x) && isPastDue(x),
       bg: 'bg-dark-200',
-      className: 'text-white',
+      className: 'text-dark',
```
Comment (cmltaWt0) [inline src/course-home/dates-tab/timeline/badgelist.jsx:55]: Is it correct?
`bg` is dark so the text should be `light`.

Comment (ihor-romaniuk) [reply src/course-home/dates-tab/timeline/badgelist.jsx:55]: Yes, it's correct.
I provide an example screenshot in the MR description and this style relates to the "Past due" label.

**Comment by itsjeyd** [comment]
Hey @cmltaWt0, it looks like @ihor-romaniuk addressed your comments. Would you be able to have another look?

Also I was wondering if you're in a position to provide final approval for this PR?

I'm asking because based on info from the [Core Contributors to the Open edX Project](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3156344833/Core+Contributors+to+the+Open+edX+Project) page seems like you're not a "regular" CC for the parent repo:

> Merge rights for all release branches
>
> Write access to `edx-app-test`, `devstack`, `edx-e2e-tests`, `cypress-e2e-tests`, `edx-app-test`
>
> For release management: write access to all repos currently in the latest named release (via [community-release-managers](https://github.com/orgs/openedx/teams/community-release-managers/repositories) group).

... And I'm not sure if reviewing and merging this PR falls under release management, if that makes sense?

Let me know what you think, it's completely possible that I'm missing something :)

**Comment by itsjeyd** [comment]
Calling all CCs for frontend-app-learning:

@brian-smith-tcril 
@arbrandes 
@viktorrusakov 
@BbrSofiane 

Would you have bandwidth for reviewing and merging this PR?

(Even if you don't, it would be great if you could comment below and state that. Otherwise it's hard to know if there's a need to keep looking for reviewers. Thanks for your help!)

**Review by brian-smith-tcril — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
@ihor-romaniuk 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-app-learning&entry.867055334=https://github.com/openedx/frontend-app-learning/pull/1223&entry.1484655318=https://github.com/ihor-romaniuk&entry.752974735=2023-10-25+12:19&entry.1917517419=2024-03-11+16:31&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**